src/js/index.js

```diff
@@ -12,9 +12,9 @@ const app = async ($targetEle) => {
   const nav = new Nav($targetEle, { leftNavBarHeadLines, rightNavBarHeadLines });
   const mainContent = new MainContent($targetEle, { pressData });
 
-  header.mount();
-  nav.mount();
-  mainContent.mount();
+  header.render();
+  nav.render();
+  mainContent.render();
```

Comment:
entry point에 이것 빼고 나머지는 다른 파일에 넣어서 안 보이게 하는 게 어떨까용 
```suggestion
  header.render();
  nav.render();
  mainContent.render();
```